foundry/applications/sceneNavigation.d.ts

```diff
@@ -14,24 +22,60 @@ declare class SceneNavigation extends Application {
    */
   get scenes(): Scene[];
 
+  /** @override */
+  render(force?: boolean, options?: Application.RenderOptions): this;
+
+  /** @override */
+  protected _render(force?: boolean, options?: Application.RenderOptions): Promise<void>;
+
+  /** @override */
+  getData(options?: Application.RenderOptions): SceneNavigation.Data | Promise<SceneNavigation.Data>;
+
+  /**
+   * Expand the SceneNavigation menu, sliding it down if it is currently collapsed
+   */
+  expand(): Promise<boolean>;
+
   /**
    * Collapse the SceneNavigation menu, sliding it up if it is currently expanded
    */
   collapse(): Promise<boolean>;
 
   /**
-   * Expand the SceneNavigation menu, sliding it down if it is currently collapsed
+   * Activate Scene Navigation event listeners
    */
-  expand(): Promise<boolean>;
+  activateListeners(html: JQuery): void;
 
   /**
-   * Prepare the default data which is required to render the SceneNavigation menu
+   * Get the set of ContextMenu options which should be applied for Scenes in the menu
    */
-  getData(): object;
+  private _getContextMenuOptions(): ContextMenu.Item[];
 
   /**
-   * Extend the Application.render logic to first check the rendering context to see what was changed
-   * If a specific context was provided, make sure an update to the navigation is necessary before rendering
+   * Handle left-click events on the scenes in the navigation menu
    */
-  render(force?: boolean, options?: Application.RenderOptions): this;
+  private _onClickScene(event: Event): void;
+
+  /** @override */
+  protected _onDragStart(event: DragEvent): void;
+
+  /** @override */
+  protected _onDrop(event: DragEvent): Promise<boolean | undefined | void>;
+
+  /** @override */
+  private _onToggleNav(event: Event): void;
```

Comment:
Event type should be `JQuery.ClickEvent`

Reply:
For the  _onDrop function, thee base Application._onDrop is expecting a DragEvent so I left that as is. Changed the onToggleNav to the JQuery Click Event.